tools-v2/pkg/config/bs.go

```diff
@@ -33,20 +33,24 @@ import (
 
 const (
 	// curvebs
-	CURVEBS_MDSADDR            = "mdsaddr"
-	VIPER_CURVEBS_MDSADDR      = "curvebs.mdsAddr"
-	CURVEBS_MDSDUMMYADDR       = "mdsdummyaddr"
-	VIPER_CURVEBS_MDSDUMMYADDR = "curvebs.mdsDummyAddr"
-	CURVEBS_ETCDADDR           = "etcdaddr"
-	VIPER_CURVEBS_ETCDADDR     = "curvebs.etcdAddr"
-	CURVEBS_PATH               = "path"
-	VIPER_CURVEBS_PATH         = "curvebs.path"
-	CURVEBS_USER               = "user"
-	VIPER_CURVEBS_USER         = "curvebs.root.user"
-	CURVEBS_DEFAULT_USER       = "root"
-	CURVEBS_PASSWORD           = "password"
-	VIPER_CURVEBS_PASSWORD     = "curvebs.root.password"
-	CURVEBS_DEFAULT_PASSWORD   = "root_password"
+	CURVEBS_MDSADDR             = "mdsaddr"
+	VIPER_CURVEBS_MDSADDR       = "curvebs.mdsAddr"
+	CURVEBS_MDSDUMMYADDR        = "mdsdummyaddr"
+	VIPER_CURVEBS_MDSDUMMYADDR  = "curvebs.mdsDummyAddr"
+	CURVEBS_ETCDADDR            = "etcdaddr"
+	VIPER_CURVEBS_ETCDADDR      = "curvebs.etcdAddr"
+	CURVEBS_PATH                = "path"
+	VIPER_CURVEBS_PATH          = "curvebs.path"
+	CURVEBS_USER                = "user"
+	VIPER_CURVEBS_USER          = "curvebs.root.user"
+	CURVEBS_DEFAULT_USER        = "root"
+	CURVEBS_PASSWORD            = "password"
+	VIPER_CURVEBS_PASSWORD      = "curvebs.root.password"
+	CURVEBS_DEFAULT_PASSWORD    = "root_password"
+	USERNAME                    = "username"
```

Comment:
Add the prefix CURVEBS. Then add VIPER_CURVEBS_FILENAME, this value indicates the path address corresponding to the flag in the configuration file, so that it can also be read from the configuration file when the configuration file provides the corresponding value. Remember to add the mapping relationship in BSFLAG2VIPER.

Reply:
what is `VIPER_CURVEBS_FILENAME` default value in the config file?
<img width="732" alt="image" src="https://user-images.githubusercontent.com/25290236/203045121-997bcbf0-0c4b-447a-af9e-9d378921d308.png">
And are `curvebs.root.user` same with `curvebs.root.username` which  I declared as `USERNAME` in line 50

Reply:
Is this ok?
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/25290236/203047837-fde0b7ba-fe46-4bf7-bb5b-0c344c21a556.png">

Reply:
> Is this ok? <img alt="image" width="1120" src="https://user-images.githubusercontent.com/25290236/203047837-fde0b7ba-fe46-4bf7-bb5b-0c344c21a556.png">


Yes, it’s the same as what you wrote on the left, but you don’t need to add the corresponding content in template.yaml. And should be: VIPER_CURVEBS_FSNAME="curvebs.fsname".

---

是的就像你左边写得一样，不过 template.yaml中可以不补充对应的内容。并且应该为： VIPER_CURVEBS_FSNAME="curvebs.fsname"。

Reply:
Then [here](https://github.com/opencurve/curve/blob/d272930d2740fde4e162b9dcab9ef560bf619ed3/tools-v2/pkg/config/bs.go#L57) should add a mapping relationship:
CURVEBS_FILENAME: VIPER_CURVEBS_FILENAME.

---

然后在[这里](https://github.com/opencurve/curve/blob/d272930d2740fde4e162b9dcab9ef560bf619ed3/tools-v2/pkg/config/bs.go#L57)应该添加一下映射关系：
CURVEBS_FILENAME: VIPER_CURVEBS_FILENAME。

Reply:
The same goes for the other options.